src/posix/net.cpp

```diff
@@ -362,25 +363,36 @@ void CurlHttpIO::addaresevents(WinWaiter *waiter)
         {
             ((WinWaiter *)waiter)->addhandle(info.handle, Waiter::NEEDEXEC);
         }
-
+#else
```

Comment:
Not sure if Windows Phone will work on the new way with Posix. To be tested and confirmed!